tests/pytorch_tests/model_tests/feature_models/const_representation_test.py

```diff
@@ -101,3 +101,39 @@ def __init__(self, unit_test):
 
     def create_networks(self):
         return ConstRepresentationMultiInputNet()
+
+
+class ConstRepresentationCodeNet(nn.Module):
```

Comment:
maybe duplicate this test to check different arguments combinations? (namely, args and kwargs, dynamic, etc.)
do you think it would be valuable to test?

Reply:
Added functional `Conv2d`, `ConvTranspose2d` & `Linear` to cover all options of `get_node_args_spec`